deploy/charts/cert-manager/values.yaml

```
@@ -109,6 +109,9 @@ serviceAccount:
 extraArgs: []
   # When this flag is enabled, secrets will be automatically removed when the certificate resource is deleted
   # - --enable-certificate-owner-ref=true
+  # This flag is deprecated, use default-secret-cleanup-strategy instead.
+  # When this flag is set to OnDelete, secrets will be automatically removed when the certificate resource is deleted
+  # - --default-secret-cleanup-strategy=OnDelete
```

I suggest mentioning the deprecation mention closer to the `--enable-certificate-owner-ref` comment.

```suggestion
  # When this flag is enabled, secrets will be automatically removed when the certificate resource is
  # deleted. This flag is deprecated, please use --default-secret-cleanup-strategy instead.
  # - --enable-certificate-owner-ref=true
  # When this flag is set to OnDelete, secrets will be automatically removed when the certificate resource is 
 deleted.
  # - --default-secret-cleanup-strategy=OnDelete
```